### PR TITLE
document building against different CUDA Toolkit versions

### PR DESCRIPTION
**Pull request by rongou**
Signed-off-by: Rong Ou <rong.ou@gmail.com>

### CODE REVIEW

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
build

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,6 +40,13 @@ mvn verify
 
 After a successful build the RAPIDS Accelerator jar will be in the `dist/target/` directory.
 
+### Building against different CUDA Toolkit versions
+
+You can build against different versions of the CUDA Toolkit by using one of the following profiles:
+* `-Pcuda10-1` (CUDA 10.1, default)
```
Comment (gerashegalov) [inline CONTRIBUTING.md:46]: is it worth documenting these profiles consisting of a single property or should we just document the property `-Dcuda.version`?

Comment (rongou) [reply CONTRIBUTING.md:46]: My original goal was to get IntelliJ to run tests using a different cuda version, adding profiles seems to be the only way to get that to work.

Comment (gerashegalov) [reply CONTRIBUTING.md:46]: I thought we could pass any property you want to a maven runner in intellij but this is probably easier

**Review by gerashegalov — APPROVED** [review]
LGTM

**Review by tgravescs — APPROVED** [review]

**Pull request merged**